tests/Application.UnitTests/Mappings/ProductMapperTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using CleanCompanyName.DDDMicroservice.Application.Products.Commands.AddProduct;
+using CleanCompanyName.DDDMicroservice.Application.Products.Commands.UpdateProduct;
+using CleanCompanyName.DDDMicroservice.Application.Products.Dto;
+using CleanCompanyName.DDDMicroservice.Domain.Entities.Products;
+using CleanCompanyName.DDDMicroservice.Domain.Entities.Products.ValueObjects;
+using FluentAssertions;
+using Mapster;
+using Xunit;
+
+namespace CleanCompanyName.DDDMicroservice.Application.UnitTests.Mappings;
+
+public class ProductMapperTests : IClassFixture<DependencyInjectionFixture>
```

Comment:
As Robert was suggesting it would be nice to test the complete mapping and not only the one we overload as the generic `Adapt` can bring surprises we don't necessarily expect. :-D